src/components/user/list.vue

```diff
@@ -208,3 +206,29 @@ export default {
   }
 }
 </style>
+
+<i18n lang="json5">
+{
+  "en": {
+    "action": {
+      "create": "Create Web User"
+    },
+    "heading": [
+      "Web Users have accounts on this website to oversee and administer the Projects on this server. Administrators can manage anything on the site. Users with no role can still be made Project Managers on any Project, from that Project’s settings.",
+      {
+        "full": "For more information, {clickHere}.",
+        "clickHere": "click here"
+      }
+    ],
+    "header": {
+      "sitewideRole": "Sitewide Role"
+    },
+    "alert": {
+      "create": "A user was created successfully for “{displayName}”.",
+      "assignRole": "Success! “{displayName}” has been given a Sitewide Role of “{roleName}”.",
+      "resetPassword": "The password for “{displayName}” has been invalidated. An email has been sent to {email} with instructions on how to proceed.",
+      "retire": "The user “{displayName}” has been retired."
```

Comment:
@lognaturel, will `assignRole` and `retire` here require gendered conjugation in Spanish? Would it help to write something like "The account for {displayName} has&hellip;" rather than "The user {displayName} has&hellip;"?

Reply:
It's not an issue in either language because of the passive construction.